Bio/SearchIO/_model/hit.py

```diff
@@ -147,26 +147,32 @@ def __init__(self, hsps=(), id=None, query_id=None):
             self.append(hsp)
 
     def __repr__(self):
+        """Return Hit id, query id, and number of hsps."""
         return "Hit(id=%r, query_id=%r, %r hsps)" % (self.id, self.query_id,
                                                      len(self))
 
     def __iter__(self):
+        """Iterate over hsps."""
         return iter(self.hsps)
 
     def __len__(self):
+        """Return number of hsps."""
         return len(self.hsps)
 
     # Python 3:
     def __bool__(self):
+        """Return True if there are hsps."""
         return bool(self.hsps)
 
     # Python 2:
     __nonzero__ = __bool__
 
     def __contains__(self, hsp):
+        """Return True if hsp in items."""
         return hsp in self._items
 
     def __str__(self):
+        """Return the string of hit and its values recursively."""
```

Comment:
I would rather state the intention of `__str__`, rather than what it does per-se.

For example, this could be something like `Returns a human readable summary of the hit object.`